sqlx-core/src/mysql/options/mod.rs

```diff
@@ -89,6 +90,7 @@ impl MySqlConnectOptions {
             ssl_ca: None,
             statement_cache_capacity: 100,
             log_settings: Default::default(),
+            disable_pipes_as_concat: true
```

Comment:
You've inverted the logic here, this will never send `PIPES_AS_CONCAT` even though that should remain the default.

The flag should just be called `pipes_as_concat` and include the flag if `true`.

It also needs a public setter so people can change it.

Reply:
Understood. Should be fixed per your suggestions.